src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -397,59 +399,70 @@
 					<FontIcon FontSize="14" Glyph="{x:Bind ViewModel.SearchButtonGlyph, Mode=OneWay}" />
 				</Button>
 
-				<Grid Visibility="{x:Bind OngoingTasksViewModel.AnyBannersPresent, Mode=OneWay}">
-					<Button
-						x:Name="OngoingTasks"
-						Padding="0"
-						AccessKey="T"
-						AutomationProperties.Name="{helpers:ResourceString Name=OngoingTasks}"
-						Style="{StaticResource AddressToolbarButtonStyle}"
-						ToolTipService.ToolTip="{helpers:ResourceString Name=OngoingTasks}">
-						<Button.Content>
-							<Grid>
-								<FontIcon
-									x:Name="OngoingTasksFontIcon"
-									x:Load="{x:Bind OngoingTasksViewModel.AnyOperationsOngoing, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
-									FontSize="16"
-									Glyph="&#xF16A;" />
-
-								<ProgressRing
-									x:Name="MedianOperationProgressRing"
-									Width="20"
-									Height="20"
-									HorizontalAlignment="Center"
-									VerticalAlignment="Center"
-									x:Load="{x:Bind OngoingTasksViewModel.AnyOperationsOngoing, Mode=OneWay}"
-									IsIndeterminate="False"
-									Value="{x:Bind OngoingTasksViewModel.MedianOperationProgressValue, Mode=OneWay}" />
-							</Grid>
-						</Button.Content>
-						<Button.Flyout>
-							<Flyout
-								contract8Present:ShouldConstrainToRootBounds="False"
-								AutomationProperties.Name="{helpers:ResourceString Name=OngoingTasks}"
-								Placement="Bottom">
-								<uc:OngoingTasksFlyout
-									x:Name="OngoingTasksControl"
-									Width="400"
-									MinHeight="300"
-									MaxHeight="500"
-									x:FieldModifier="public"
-									IsTabStop="True"
-									ViewModel="{x:Bind OngoingTasksViewModel}" />
-							</Flyout>
-						</Button.Flyout>
-					</Button>
-					<InfoBadge
-						x:Name="StatusInfoBadge"
-						HorizontalAlignment="Right"
-						VerticalAlignment="Top"
-						Value="{x:Bind OngoingTasksViewModel.InfoBadgeValue, Mode=OneWay}" />
-				</Grid>
-
-				<!--  Install Update Notification  -->
+				<!--  Statuc Center  -->
 				<Button
-					x:Name="UpdateButton"
+					x:Name="ShowStatusCenterButton"
+					Padding="0"
+					AccessKey="T"
+					AutomationProperties.Name="{helpers:ResourceString Name=OngoingTasks}"
+					Style="{StaticResource AddressToolbarButtonStyle}"
+					ToolTipService.ToolTip="{helpers:ResourceString Name=OngoingTasks}"
+					Visibility="{x:Bind OngoingTasksViewModel.HasAnyItem, Mode=OneWay}">
+
+					<Grid Margin="-16">
```

Comment:
This can be wrapped in `<Button.Content>`. As a side note, what's the negative margin for?

Reply:
Though I set button paddint to zero, the button still have 16 spacings for unknown reasons. As a workaround, I set grid margin to -16px. do you know why?

Reply:
I rebuild the solution but no changes there.

Reply:
Okay then.